Gradle Support

AndrewReitz opened the pull request:
Updated Project to use Gradle Support and Added the Ability to easily push an aar to maven central

AndrewReitz committed:
added gradle files

AndrewReitz committed:
Removed Checkstyle

Since Gradle Doesn't Support It

AndrewReitz committed:
Updating Project Structure

Updating Project Structure for Gradle

AndrewReitz committed:
Updated Gradle Builds

Allowing for easy publishing to central

AndrewReitz committed:
Updating version numbers

AndrewReitz committed:
Removing Poms

AndrewReitz committed:
replaced groupId

replaced test groupId for publishing snapshot

AndrewReitz committed:
Updated build.gradle

fixed not publishing based on param passed in

AndrewReitz committed:
Improved Upload to Central

 Updated how uploads to central are done to improve the processes and added signing, source and javadocs

AndrewReitz committed:
Easier Usage

AndrewReitz committed:
Fixed usage

trippedout commented:
any luck with this? getting lots of errors with gradle and support lib clashing

AndrewReitz commented:
@trippedout The support library just updated again.  I will get another publish out asap.

AndrewReitz commented:
Updated, I am going to close this pull since I don't think it's going to get pulled in. You can get the one I updated here https://github.com/InkApplications/Android-ViewPagerIndicator
